phalcon/Storage/Adapter/Redis.zep

```diff
@@ -15,10 +15,9 @@ use Phalcon\Storage\Adapter\AbstractAdapter;
 use Phalcon\Storage\Exception;
 use Phalcon\Storage\SerializerFactory;
 use Phalcon\Storage\Serializer\SerializerInterface;
+use Redis;
```

Comment:
I don't think this will work since the actual class is called Redis. Perhaps add an alias here? This was the reason that I left the `\` in the connection class below.